src/Microsoft.FeatureManagement/FeatureFilters/TimeWindowFilter.cs

```diff
@@ -33,7 +37,26 @@ public TimeWindowFilter(ILoggerFactory loggerFactory)
         /// <returns><see cref="TimeWindowFilterSettings"/> that can later be used in feature evaluation.</returns>
         public object BindParameters(IConfiguration filterParameters)
         {
-            return filterParameters.Get<TimeWindowFilterSettings>() ?? new TimeWindowFilterSettings();
+            DateTimeOffset? Start = filterParameters["Start"] != null ? filterParameters.GetValue<DateTimeOffset>("Start") : (DateTimeOffset?)null;
+            DateTimeOffset? End = filterParameters["End"] != null ? filterParameters.GetValue<DateTimeOffset>("End") : (DateTimeOffset?)null;
+
+            TimeWindowFilterSettings settings = new TimeWindowFilterSettings()
+            {
+                Start = Start,
+                End = End
+            };
+
+            List<string> Filters = filterParameters.GetSection("Filters").Get<List<string>>();
+            if (Filters != null)
+            {
+                foreach (string expression in Filters)
+                {
+                    CrontabExpression crontabExpression = CrontabExpression.Parse(expression);
+                    settings.Filters.Add(crontabExpression);
```

Comment:
I would recommend to build the filters than assign it to settings, rather than assuming initial state of settings Filters.

Reply:
I moved the code of parsing Cron to the EvaluateAsync() instead if BindParameters().
Now, the TimeWindowFilterSettings will store the raw string of Crons.